src/domain/participant/index.js

```diff
@@ -447,16 +447,53 @@ const getLimitsForAllParticipants = async ({ currency = null, type = null }) =>
  * @returns {integer} - Returns number of database rows affected if successful, or throws an error if failed
  */
 
+// {
```

Comment:
Please, avoid adding commented out code. Either update the example above, that is part of jsdoc or add another example, if different